geomstats/geometry/discrete_curves.py

```diff
@@ -326,6 +331,47 @@ def square_root_velocity_inverse(self, srv, starting_point):
 
         return curve
 
+    def d_square_root_velocity(self, tangent_vec, curve):
```

Comment:
For consistency with our current namings of differentials, this should be called:
`differential_square_root_velocity`, even if it's a bit long...

See .e.g here:
https://github.com/geomstats/geomstats/blob/14f181ae9d801c729037698e8926587c9f83163b/geomstats/geometry/spd_matrices.py#L133